icons/ngrx/ngrx-plain.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M64.1 126.6l-.1-.1-.1.1v-.1L14 99.3 5.6 21.7 63.9 1.5v-.1h.2v.1l58.3 20.2-8.4 77.6-49.9 27.2zm29.1-82.3l-.1-.1.1.1zm0 0q4.9 17.8-12.1 31.1Q87.4 76.5 93 72q-9.8 10-20.8 10.3 4.3 4.9 11.4 4.7-16.3 3.8-29.1-11.4c-.6 2.8 1.2 6.2 5.3 10.2q-12.7-5.9-12.7-19-.2-14.2 14.9-18.5 28.8 11.9 30.2.6Q91 40.3 78.6 33q-8.7-5-17.1-4.5-3.4-3.5-7.8-3.3c-2 0-3.6.6-4.7 1.8-1.7 1.8-4.6 5.7-9.8 6.1q-5.3.4-5.2 5.1.1 4.8-.4 5.4-.5 1.1-3.1 3.3-2.6 2.1-2.6 4.1.2 4 1.5 5.9.6 2.8-.5 4.4c-1 1.5-1.6 2-1.3 3.6.4 1.6.4 2 4.1 5.9q3.4 10.8 12.5 17.9s9.5 8.8 24.3 8.8q22.9-1.1 31.4-21.7-8 6-12.9 6.4 15.3-8.6 13.5-25.1-4.2 8.1-7.4 10.6 4.3-6.8 4.2-12.9-.2-6-4.1-10.5zm-20.6-1.4c-.9 0-1.7-.8-1.7-1.7 0-.9.8-1.6 1.7-1.6.9 0 1.6.7 1.6 1.6 0 .9-.7 1.7-1.6 1.7z"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 128 128"><path fill="#000" d="M49.64 0 3.489 21.34l4.326 68.566Zm28.72 0 41.78 89.906 4.372-68.566zM66.38 24.678a4.39 4.39 0 0 0-1.566.32 39.575 39.575 0 0 0-.644.272c-2.11.903-5.613 2.403-9.516.921-2.887-1.096-4.914-.394-6.08 2.108-1.148 2.566-1.918 3.959-2.308 4.18-.594.462-1.856.946-3.778 1.44-1.92.496-3.126 1.26-3.615 2.294-.902 2.19-1.229 3.96-.98 5.305-.433 1.668-1.128 2.794-2.086 3.353a28.706 28.706 0 0 1-.637.358c-1.004.553-1.506.83-1.758 2.013l-.02.096c-.284 1.332-.38 1.773.99 6.213-.894 6.585.193 12.928 3.259 19.004-.004-.008 4.287 10.71 16.091 16.297 12.504 5.17 23.636 3.337 33.397-5.498-5.783 1.165-10.04 1.246-12.75.257 10.307-.752 17.052-5.777 20.271-15.07-4.286 3.31-7.606 5.238-9.935 5.758 3.99-2.543 6.72-5.474 8.209-8.774 1.489-3.299 1.728-6.645.717-10.035l-.004.031c-1.848 10.767-8.994 17.572-21.438 20.418 3.078 2.157 6.672 2.742 10.832 1.764-7.769 2.88-14.622 2.988-20.58.346 1.048 3.71 3.505 6.386 7.41 8.068-9.68-2.078-16.026-8.78-19.027-20.101-1.532 2.022-1.418 5.394.34 10.171-5.241-6.344-6.217-13.013-2.91-19.992 3.466-7.67 9.763-10.744 18.892-9.218C79.55 60.574 87.54 64.545 91.09 58.873c1.553-4.851-.044-10.82-4.8-17.883-3.417-4.825-7.442-8.21-12.038-10.1-.941-2.67-2.614-4.54-4.963-5.519-1.023-.477-1.997-.714-2.908-.693zM77.909 41.82s-.023 2.846.856 4.041c.878 1.196 2.003 3.188-.524 1.596-2.526-1.592-1.273-4.42-.332-5.637zM34.213 99.51 32.43 110 64 128l31.57-18-1.78-10.49Z"/></svg>
```

Comment:
This can be simplified a bit more by removing the fill
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path d="M49.64 0 3.489 21.34l4.326 68.566Zm28.72 0 41.78 89.906 4.372-68.566zM66.38 24.678a4.39 4.39 0 0 0-1.566.32 39.575 39.575 0 0 0-.644.272c-2.11.903-5.613 2.403-9.516.921-2.887-1.096-4.914-.394-6.08 2.108-1.148 2.566-1.918 3.959-2.308 4.18-.594.462-1.856.946-3.778 1.44-1.92.496-3.126 1.26-3.615 2.294-.902 2.19-1.229 3.96-.98 5.305-.433 1.668-1.128 2.794-2.086 3.353a28.706 28.706 0 0 1-.637.358c-1.004.553-1.506.83-1.758 2.013l-.02.096c-.284 1.332-.38 1.773.99 6.213-.894 6.585.193 12.928 3.259 19.004-.004-.008 4.287 10.71 16.091 16.297 12.504 5.17 23.636 3.337 33.397-5.498-5.783 1.165-10.04 1.246-12.75.257 10.307-.752 17.052-5.777 20.271-15.07-4.286 3.31-7.606 5.238-9.935 5.758 3.99-2.543 6.72-5.474 8.209-8.774 1.489-3.299 1.728-6.645.717-10.035l-.004.031c-1.848 10.767-8.994 17.572-21.438 20.418 3.078 2.157 6.672 2.742 10.832 1.764-7.769 2.88-14.622 2.988-20.58.346 1.048 3.71 3.505 6.386 7.41 8.068-9.68-2.078-16.026-8.78-19.027-20.101-1.532 2.022-1.418 5.394.34 10.171-5.241-6.344-6.217-13.013-2.91-19.992 3.466-7.67 9.763-10.744 18.892-9.218C79.55 60.574 87.54 64.545 91.09 58.873c1.553-4.851-.044-10.82-4.8-17.883-3.417-4.825-7.442-8.21-12.038-10.1-.941-2.67-2.614-4.54-4.963-5.519-1.023-.477-1.997-.714-2.908-.693zM77.909 41.82s-.023 2.846.856 4.041c.878 1.196 2.003 3.188-.524 1.596-2.526-1.592-1.273-4.42-.332-5.637zM34.213 99.51 32.43 110 64 128l31.57-18-1.78-10.49Z"/></svg>
```